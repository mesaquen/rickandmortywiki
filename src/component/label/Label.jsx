import React, { memo } from 'react';
import styled from 'styled-components';
import Text from '../text/Text';

const Container = styled.div`
  display: flex;
  justify-content: space-between;
`;

function Label(props) {
  const { title, children } = props;
  return (
    <Container>
      <Text color="#999">{title}</Text>
      <Text color="teal">{children}</Text>
    </Container>
  );
}

export default memo(Label);
