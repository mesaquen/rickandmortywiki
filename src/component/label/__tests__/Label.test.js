import React from 'react';
import Renderer from 'react-test-renderer';
import Label from '../Label';
/**
 * @author Mesaque Francisco <mesaquenf@gmail.com>
 * @since 2019-11-16
 */

describe('Label', () => {
  let component = Renderer.create(<Label title="title">Value</Label>);
  it('should have same snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
