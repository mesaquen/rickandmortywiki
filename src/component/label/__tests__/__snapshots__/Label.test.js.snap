// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should have same snapshot 1`] = `
<div
  className="sc-kAzzGY ftdBxt"
>
  <span
    className="sc-htpNat SwWzv"
    color="#999"
    size="16px"
  >
    title
  </span>
  <span
    className="sc-htpNat kNOgIk sc-chPdSV cMhbop"
    color="teal"
    size="16px"
  >
    Value
  </span>
</div>
`;
